paasta_tools/cli/utils.py

```diff
@@ -524,7 +528,11 @@ def execute_paasta_serviceinit_on_remote_master(subcommand, cluster, service, in
         return (
             'ERROR: could not find connectable master in cluster %s\nOutput: %s' % (cluster, output)
         )
-    return run_paasta_serviceinit(subcommand, master, service, instances, cluster, stream, **kwargs)
+    if ignore_ssh_output:
```

Comment:
It's because of the streaming mode, it sends a `-t` flag to ssh which makes it print out those extra messages. Yeah I guess the name is kinda misleading.

>Any particular reason not to just do QUIET all the time?

I feel like having all of SSH's output is useful for commands like an emergency stop, just in case. (they all share this same code path)